cmd/compose.go

```diff
@@ -110,62 +112,62 @@ func compose() {
 	prepConfig()
 
 	// Generate code to build the plugin according to the config file
-	log.Info("generating plugin")
+	LogWithCommand.Info("generating plugin")
 	generator, err := p2.NewGenerator(genConfig, databaseConfig)
 	if err != nil {
-		log.Fatal(err)
+		LogWithCommand.Fatal(err)
 	}
 	err = generator.GenerateExporterPlugin()
 	if err != nil {
-		log.Debug("generating plugin failed")
-		log.Fatal(err)
+		LogWithCommand.Debug("generating plugin failed")
```

Comment:
I dig this debug statement 👍 

Maybe worth adding similar ones around other `Fatal` logs? e.g. if initializing the generator or getting the plugin paths fails